### ISSUE TITLE
registerApexTestWireAdapter is not emitting data and unit testing are failing as a result

### ISSUE THREAD

**Issue by adityavgpl** [salesforce/wire-service-jest-util#25]
I have following code for LWC unit test
```
import { createElement } from 'lwc';
import MyComponent from 'c/myComponent';
import getData '@salesforce/apex/FormController.getData';
import { registerApexTestWireAdapter } from '@salesforce/sfdx-lwc-jest';

const mockRecords = require('./data/getRecords.json');
const getDataAdapter = registerApexTestWireAdapter(getData);

describe('c-my-component', () => {
    afterEach(() => {
        while (document.body.firstChild) {
            document.body.removeChild(document.body.firstChild);
        }

        jest.clearAllMocks();
    });

    it('renders component based on data', () => {
        const element = createElement('c-my-component', {
            is: MyComponent 
        });
        document.body.appendChild(element);

        getDataAdapter.emit(mockRecords);

        return Promise.resolve().then(() => {
            const listEls = element.shadowRoot.querySelectorAll(
                '.list-element'
            );

            expect(listEls.length).toBe(mockRecords.length);
       });
    });
```
This unit test is failing because the **apex test wire adaptor** is not emitting data and the test component is not rendering list in jsdom.

In my application the "**@salesforce/wire-service-jest-util**"  version was **2.4.0** in package-lock.json, However when I downgraded the version to **2.3.0** the wire adaptor was emitting data properly and unit test cases were working fine.

Is this an known issue with **@salesforce/wire-service-jest-util** version 2.4.0, Is there any way by which we can use **ApexTestWireAdaptor** for emitting data while using **wire-service-jest-util 2.4.0** ?

**Comment by trevor-bliss** [comment]
@jodarove

**Comment by jodarove** [comment]
Thanks for reporting this @adityavgpl .

The issue was introduced in PR #22 , which has a bug that changes format of the value emitted to the component.

#26 fixes the issue.

About:
> Is this an known issue with @salesforce/wire-service-jest-util version 2.4.0, Is there any way by which we can use ApexTestWireAdaptor for emitting data while using wire-service-jest-util 2.4.0 ?

Yes, you could use:
```js
getDataAdapter.emit({ data: mockRecords, error: undefined });
```
But notice that with the fix (probably v2.4.1), your test will be broken again.

**Comment by adityavgpl** [comment]
Awesome. Thanks a lot @jodarove and @trevor-bliss for your quick response.

**Comment by jodarove** [comment]
fixed with #26 (v2.4.1)

**Comment by jodarove** [comment]
please use v2.4.2, since 2.4.1 is faulty.
